src/components/atoms/ModalIcon.js

```diff
@@ -0,0 +1,15 @@
+import { Box } from "@mui/material";
+
+function ModalIcon({icon}) {
```

Comment:
should be something like this
`function ModalIcon(props){`

then before the return you destructure it
`const {icon} = props`
just in case we need more props later on

Reply:
Done